### DISCUSSION TITLE
Add pulley ratio to equipment

### DISCUSSION THREAD

**Discussion by astashov** [astashov/liftosaur#168] [Ideas]
From Discord:

> I go to 2 gyms. The cable machine at one gym has a 2:1 pulley ratio and the other is 1:1. Any recommendations on how to handle this in the app? Right now I'm just mentally halving the weight stack on the 2:1 machine. The arithmetic is taking too many calories away from my gains.

**Comment by yrinkel** [comment]
just to mention, exactly the same idea was already opened a while ago (#142) see here:
https://github.com/astashov/liftosaur/discussions/142#discussion-7096916

**Comment by astashov** [comment]
Duplicate of https://github.com/astashov/liftosaur/discussions/142
